<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveCuttingDialog</class>
 <widget class="QDialog" name="InteractiveCuttingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Cutting as in &quot;cutting video&quot;">Cutting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frmPlay">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="layoutPlayer"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutRangeSelector"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmControl">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="btnAsBegin">
          <property name="text">
           <string>Mark as Begin</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/actions/icons/mark.svg</normaloff>:/actions/icons/mark.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnToBegin">
          <property name="text">
           <string>Seek to Begin</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/actions/icons/seek_to_begin.svg</normaloff>:/actions/icons/seek_to_begin.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlaySelection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Play Selection</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/actions/icons/preview.svg</normaloff>:/actions/icons/preview.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnAsEnd">
          <property name="text">
           <string>Mark as End</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/actions/icons/mark.svg</normaloff>:/actions/icons/mark.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnToEnd">
          <property name="text">
           <string>Seek to End</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/actions/icons/seek_to_end.svg</normaloff>:/actions/icons/seek_to_end.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmEdit">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="layoutRangeEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InteractiveCuttingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InteractiveCuttingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
